src/Middleware/StaticFiles/src/PublicAPI.Unshipped.txt

```diff
@@ -1,58 +1 @@
 #nullable enable
-Microsoft.AspNetCore.Builder.DefaultFilesOptions.DefaultFileNames.get -> System.Collections.Generic.IList<string!>!
```

Comment:
Accidental file change?

Reply:
All APIs were in Shipped and Unshipped. I think there was a bad merge at some point.